packages/lnlambda/pubspec.yaml

```diff
@@ -16,7 +17,8 @@ dev_dependencies:
   test: ^1.16.0
   build_runner: ^2.0.0
   json_serializable: ^6.2.0
-  clightning_rpc: ^0.0.2-beta.1
+  clightning_rpc:
+    path: ../rpc
```

Comment:
if you use the override dependencies here you can remove the `publish_to: none`?

Reply:
I am getting this warning in this case `Publishable packages can't have 'path' dependencies.`